README.md

```diff
@@ -23,6 +23,45 @@ Or install it yourself as:
 
 TODO: Write usage instructions here
 
+
+### TLS
+
+The registry does support client TLS cerificates as a means of authenticating w/ the schema registry.  
+
+You have two options of passing options to net/http.  You can pass in all the config as named parameters
+
+##### Example
+
+```ruby
+    @client = SchemaRegistry::Client.new(
+        "https://testschmreg.domain.com:8082",
+        '',
+        '', 
```

Comment:
Note: you can leave those two arguments out because they are optional:

``` ruby
@client = SchemaRegistry::Client.new(
  "https://testschmreg.domain.com:8082",
  cert: ...,
  key: ...,
)
```

They will be set to `nil`